docs/development.md

```diff
@@ -320,19 +320,19 @@ To use this box, copy the configuration from `boxes.yaml.example` to
 `boxes.yaml`, changing options as necessary, then run the following:
 
 ```sh
-vagrant up centos7-dynflow-devel
+vagrant up centos9-dynflow-devel
```

Comment:
this box doesn't exist -- there is only an alma8 one -- care to update that definition to cs9 in local.example?